### PR TITLE
Test Register page

### PR DESCRIPTION
**Pull request by uyen3893**
Please review it, thanks.

### CODE REVIEW

File: test/pageobjects/secure.page.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const Page = require('./page');
```
Comment (phkhuong) [inline test/pageobjects/secure.page.js]: Please remove example file.

File: test/specs/example.e2e.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const LoginPage = require('../pageobjects/login.page');
```
Comment (phkhuong) [inline test/specs/example.e2e.js]: Please remove example file.

File: test/pageobjects/page.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+/**
+* main page object containing all methods, selectors and functionality
```
Comment (phkhuong) [inline test/pageobjects/page.js]: Please remove example file.

File: test/specs/parameter.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+const firstname = 'uyen'
+const lastname = 'nguyen'
+const email = 'adain.48@dikybuyerj.com'
+const invalid_email ='test@email.com'
+const invalid_value = 'acb'
+const password = 'Test@123'
+const blank_value = ''
+var character = ''
```
Comment (phkhuong) [inline test/specs/parameter.js]: This file's purpose is constant declarations. Don't declare a variable here that could be changed somewhere else.

File: test/specs/test_register_page.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        })
+    
+        it('3. should show error message when inputting invalid email', () => {
+            RegisterPage.register(parameter.blank_value, parameter.blank_value, parameter.invalid_value, parameter.blank_value)
+            expect(RegisterPage.email_message).toHaveText('Wrong email')
+        })
+    
+        it('4. should show error message when inputting password equal or less than 5 characters', () => {
+            for(var i = 1; i <= 5; i++) {
+                parameter.character += 'a'
```
Comment (phkhuong) [inline test/specs/test_register_page.js]: It's fine to just test one case.

**Review by phkhuong — CHANGES_REQUESTED** [review]

File: test/specs/test_register_page.js (hunk 5 above)
Comment (uyen3893) [reply test/specs/test_register_page.js]: I fixed it

File: test/specs/parameter.js (hunk 4 above)
Comment (uyen3893) [reply test/specs/parameter.js]: I fixed it

File: test/pageobjects/page.js (hunk 3 above)
Comment (uyen3893) [reply test/pageobjects/page.js]: I fixed it

File: test/specs/example.e2e.js (hunk 2 above)
Comment (uyen3893) [reply test/specs/example.e2e.js]: I fixed it

File: test/pageobjects/secure.page.js (hunk 1 above)
Comment (uyen3893) [reply test/pageobjects/secure.page.js]: I fixed it

**Review by phkhuong — APPROVED** [review]

**Pull request merged**